lib/webpacker/helper.rb

```diff
@@ -96,6 +96,9 @@ def favicon_pack_tag(name, **options)
   #   <%= javascript_pack_tag 'calendar' %>
   #   <%= javascript_pack_tag 'map' %>
   def javascript_pack_tag(*names, **options)
+    raise "To prevent duplicated chunks on the page, you should call javascript_pack_tag only once on the page." if defined?(@javascript_pack_tag_loaded)
```

Comment:
Instead of raising an error that says "you shouldn't do this", is it possible to link to some kind of documentation that explains why this restriction is in place and a couple of ways the developer can resolve this? For instance, the new section that is getting added to the Usage section in the README. This is a change in behavior from the existing behavior in v5, so my fear is that even people who have been keeping up with the changes in the beta will be frustrated upon receiving this error.

Reply:
I agree. This is a fairly strong reaction to something that will not crash an app, but only cause rendering issues. 

For the record we could link to the section added by https://github.com/rails/webpacker/pull/3084.

Reply:
@guillaumebriday How about if we **raise** by default for Rails.env.test? and Rails.env and warn otherwise?